pkg/evaluators/authorization/authzed.go

```diff
@@ -60,10 +61,11 @@ func (a *Authzed) Call(pipeline auth.AuthPipeline, ctx gocontext.Context) (inter
 	if err != nil {
 		return nil, err
 	}
+	permissionStr, err := json.StringifyJSON(permission)
```

Comment:
Do we care for doing any handling for the `err` here? Since nothing is done with it, it just gets reassigned in the next line 🤔

Reply:
Fixed.